app/assets/stylesheets/content/work_packages/_table_content.sass

```diff
@@ -135,6 +135,12 @@ html:not(.-browser-mobile)
   .inline-edit--display-field
     display: initial
 
+    > span:first-of-type
```

Comment:
This will affect **every** display field with a span in it. So for example, single User fields or multi-select List fields are affected, too. I don't think that we want to reduce the width of these fields unnecessarily.

Reply:
@HDinger Your are right. Thanks for the hint with the 'custom-option' class. I wrapped this class around the other items and the styling worked out very well. I also removed the appending of the '...' which I found a bit strange considering that we have styles for this.
And I noticed, that the badge would only be shown if there are more than two items in a list. I hope this was not somehow a wanted behaviour because I changed that as well.